.github/workflows/dapr-perf.yml

```diff
@@ -231,11 +226,15 @@ jobs:
         run: |
           az acr login --name ${{ env.TEST_PREFIX }}acr
         shell: bash
-      - name: Push all container images to the registry
+      - name: Push container images
```

Comment:
Can we call this `Push Dapr container images` or something more specific than just `container`? I was confused at first as this push is happening before we build the apps below.

Reply:
Done